wdl2cwl/tests/cwl_files/smoove.cwl

```diff
@@ -25,6 +27,11 @@ requirements:
             --fasta $(inputs.referenceFasta.path) \
             $(inputs.bamFile.path)
   - class: InlineJavascriptRequirement
+hints:
```

Comment:
Why `hints` instead of `requirements`? The WDL `runtime` directive is not optional: https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#runtime-section

Reply:
Ok. I will move it in to requirements